YgoProPatcher/Client.Designer.cs

```diff
@@ -189,16 +189,16 @@ private void InitializeComponent()
             this.internetCheckbox.Name = "internetCheckbox";
             this.internetCheckbox.Size = new System.Drawing.Size(383, 24);
             this.internetCheckbox.TabIndex = 11;
-            this.internetCheckbox.Text = "Download HQ Pics from the Internet if possible";
+            this.internetCheckbox.Text = "Download high quality pictures from the internet if possible";
```

Comment:
I don’t think this will fit on button, the text is too long

Reply:
I think it will fit.